### ISSUE TITLE
Animations doesn't update in the add-on

### ISSUE THREAD

**Issue by DShark96** [bridge-core/editor#944]
**Summary**

**To Reproduce**
Just set an animation, then change it in blockbench and update in the modpack. In game you will see the old one

**Platform (please complete the following information):**

-   OS: Windows 11 22H2 Build: 22621.1992
-   App Version: 2.6.1

**Video**

https://github.com/bridge-core/editor/assets/18628885/bca6beca-73da-4bf3-8f03-95f67610872f

**Comment by ThomasOrs** [comment]
Can you check the watchmode logs? It might be that bridge. is not recompiling on save from blockbench

**Comment by outercloudstudio** [comment]
Yeah, bridge is probably not compiling it automatically. Try refreshing the project or restarting watch mode.
